packages/doenetml-worker/lib-doenetml-core/src/dast/ref_expand.rs

```diff
@@ -237,26 +265,31 @@ impl Expander {
                 element.children.push(flat_root.merge_content(
                     &DastElementContent::Error(DastError {
                         message: "Invalid `extend` attribute".to_string(),
-                        position: extend.position.clone(),
+                        position: extend_or_copy.position.clone(),
                     }),
-                    extend.parent,
+                    extend_or_copy.parent,
                 ));
 
                 // We took this memory earlier, so we need to put it back.
                 flat_root.nodes[i] = node;
                 continue;
             }
 
-            // Since this reference was inside an `extend` attribute,
+            // Since this reference was inside an `extend` or `copy` attribute,
             // we must mark it as such
             // (which indicates that we shouldn't change the component type
             // from the one specified in the element)
-            element.extending = extend_referent.map(|source| source.as_attribute());
+            if extend_or_copy.name == "copy" {
```

Comment:
Earlier in the code we were doing a case-insensitive match. I forget of the JS normalization lowercases all the attributes...but we should stay consistent I think.

Reply:
Apparently the case is not normalized, as alternative case attributes break if I remove the case-insensitive comparison.